internal/v5/api.go

```diff
@@ -235,6 +235,7 @@ func RouterHandlers(h *ReqHandler) *router.Handlers {
 			"perm":             h.puttableBaseEntityHandler(h.metaPerm, h.putMetaPerm, "acls", "developmentacls"),
 			"perm/":            h.puttableBaseEntityHandler(h.metaPermWithKey, h.putMetaPermWithKey, "acls", "developmentacls"),
 			"promulgated":      h.baseEntityHandler(h.metaPromulgated, "promulgated"),
+			"resources":        h.EntityHandler(h.metaResources, "_id", "charmmeta"),
```

Comment:
This line is strictly for meta-get.  The doc comment below relates to the non-meta endpoint (which incidentally does not  actually handle PUT).